src/quo2/components/settings/settings_item/style.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.settings.settings-item.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn find-icon-height
+  [description tag image]
+  (let [icon-height (if (= image :icon-avatar) 32 20)
+        icon-height (if description 40 icon-height)]
```

Comment:
Maybe use `cond` here

Reply:
Can you elaborate how? Here we first set the height based on what type of `image`. Then we add more height based on description. Then we may add more height based on tag.